components/src/Input/Input.js

```diff
@@ -26,12 +28,14 @@ const getInputStyle = props => {
   return inputStyles.default;
 };
 
-const getColor = props => getInputStyle(props).color;
+const getTextColor = props => getInputStyle(props).textColor;
 const getBorderColor = props => getInputStyle(props).borderColor;
+const getBackgroundColor = props => getInputStyle(props).backgroundColor;
 
 const Input = styled.input`
     width: 100%
-    color: ${getColor};
+    color: ${getTextColor};
+    background-color: ${getBackgroundColor};
     border: 1px solid;
     border-color: ${getBorderColor};
     border-radius: ${theme.radii.medium};
```

Comment:
The border radius doesn't seem to be working. Maybe @matthewlyle knows how to fix?

I get the following when starting storybooks, and the input border is square:
```
WARNING in ./src/theme.js 40:12-39
"export 'radius_border_circle' (imported as 'tokens') was not found in '@nulogy/tokens'
 @ ./.storybook/config.js
 @ multi ../node_modules/@storybook/core/dist/server/common/polyfills.js ../node_modules/@storybook/core/dist/server/preview/globals.js ./.storybook/config.js ../node_modules/webpack-hot-middleware/client.js?reload=true
```

Reply:
a merge from master should fix this. me and christiaan ran into this earlier and merged this to solve it https://github.com/nulogy/design-system/pull/178

Reply:
Yeah this is solved now.